chains/evm/calls/bridge.go

```diff
@@ -269,3 +269,23 @@ func IsProposalVotedBy(evmCaller ContractCallerClient, by common.Address, p *pro
 	out0 := *abi.ConvertType(res[0], new(bool)).(*bool)
 	return out0, nil
 }
+
+// Public function to generate bytedata for adminWithdraw contract method
+// Used to manually withdraw funds from ERC safes
+// @dev data is ABI-encoded withdrawal parameters specific to the handler
+// https://github.com/ChainSafe/chainbridge-solidity/blob/anderson/add-erc1155-support/contracts/Bridge.sol#L304-L315
+func PrepareWithdrawInput(handlerAddress common.Address, data []byte) ([]byte, error) {
```

Comment:
Some unit tests for this would be nice as we don't know now if the ABI maybe changes if this still works.